website/docs/docs/deploy/continuous-integration.md

```diff
@@ -50,3 +50,6 @@ When you push a new commit to a PR, dbt Cloud enqueues a new CI run for the late
 
 <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/example-smart-cancel-job.png" width="70%" title="Example of an automatically canceled run"/>
 
+### Run slot treatment
```

Comment:
non blocker suggestion: took a quick look at the staging for [this section](https://docs-getdbt-com-git-ly-docs-ci-jobs-dbt-labs.vercel.app/docs/deploy/continuous-integration#differences-between-ci-jobs-and-other-deployment-jobs) and it looks like it'd be worth while to have a high-level/overview table or something visualizing the differences.